### PR TITLE
Adds COMMON_EDXAPP_SETTINGS

### PR DESCRIPTION
**Pull request by pomegranited**
This change allows us to use the `openstack` or other settings files when ansible runs `manage.py` tasks, e.g. by setting:

```yaml
COMMON_EDXAPP_SETTINGS: 'openstack'
```

* Default value is 'aws' (unchanged)
* Used by `EDXAPP_SETTINGS`, `gitreload_env.DJANGO_SETTINGS`, and `demo_edxapp_settings` variables

**JIRA tickets**: [OSPR-1642](https://openedx.atlassian.net/browse/OSPR-1642)

**Sandbox URL**: 

* LMS: http://ficus.sandbox.opencraft.hosting/
* Studio: http://studio.ficus.sandbox.opencraft.hosting/

This sandbox was configured using the [open-craft:jill/ficus.rc1](https://github.com/edx/configuration/compare/master...open-craft:jill/ficus.rc1) branch, which contains the changes from this PR.

**Merge deadline**: ASAP - we'd like to get this into the ficus RC1 cut.  CC @nedbat 

Configuration Pull Request
---

Make sure that the following steps are done before merging

  - [x] @mtyaka has commented with :+1:
  - [ ] @devops team member has commented with :+1:
  - [-] are you adding any new default values that need to be overridden when this goes live?  
    - [ ] Open a ticket (DEVOPS) to make sure that they have been added to secure vars.
    - [ ] Add an entry to the CHANGELOG.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @pomegranited! It looks like you're a member of a company that does contract work for edX. If you're doing this work as part of a paid contract with edX, you should talk to edX about who will review this pull request. If this work is not part of a paid contract with edX, then you should ensure that there is an OSPR issue to track this work in [JIRA](https://openedx.atlassian.net), so that we don't lose track of your pull request.

<a   href="https://openedx-webhooks.herokuapp.com/github/process_pr?repo=edx%2Fconfiguration&number=3601"   rel="nofollow" >Create an OSPR issue for this pull request</a>.

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @pomegranited! I've created [OSPR-1642](https://openedx.atlassian.net/browse/OSPR-1642) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).



If you like, you can add yourself to the [AUTHORS](https://github.com/edx/configuration/blob/master/AUTHORS)      file for this repo, though that isn't required.     Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst)     file for more information.

File: playbooks/roles/common_vars/defaults/main.yml [hunk 1]
```diff
@@ -11,6 +11,9 @@ COMMON_BASIC_AUTH_EXCEPTIONS:
   - 192.168.0.0/16
   - 172.16.0.0/12
 
+# Use settings to use for calls to edxapp manage.py
```
Comment (mtyaka) [inline playbooks/roles/common_vars/defaults/main.yml]: typo: "Use settings to use..."

Comment (pomegranited) [reply playbooks/roles/common_vars/defaults/main.yml]: Thanks for spotting that, @mtyaka :)  Fixed with 336b6f2.

**Comment by mtyaka** [comment]
:+1:

- [x] I tested this: Successfully deployed an instance using edx_sandbox playbook on openstack.
- [x] I read through the code
- [ ] Includes documentation (no; should we add a note about it to [the changelog](https://github.com/edx/configuration/blob/master/CHANGELOG.md)?)

**Comment by pomegranited** [comment]
@mtyaka I wasn't sure whether it needed to be mentioned in the Changelog, because the default value (`aws`) has not changed.  Should I update the Changelog anyway?

**Comment by mtyaka** [comment]
@pomegranited I'm not sure what the cahangelog policy is, but I think it would be good to add even though it's backwards compatible. Adding something like "Added `COMMON_EDXAPP_SETTINGS` (default: `"aws"`)" would be enough, I think.

**Comment by pomegranited** [comment]
@mtyaka Done! 3d20ebe

**Comment by mtyaka** [comment]
Thank you : )

**Comment by pomegranited** [comment]
Hi @edx/devops - we'd like to get this change, plus https://github.com/edx/configuration/pull/3602 and https://github.com/edx/configuration/pull/3606, into `ficus.master`.  

Is there anything we can do to get the process moving?

**Review by feanil — APPROVED** [review]
@edx/devops this change looks good to me. Can I get a second review?

**Review by maxrothman — APPROVED** [review]

**Comment by feanil** [comment]
@pomegranited feel free to merge.

**Comment by pomegranited** [comment]
Yay, thank you @maxrothman and @feanil !

**Comment by pomegranited** [comment]
@nedbat 86dc3a2 needs cherry-picking into `open-release/ficus.master`.  Should I create another PR, or do I need someone on your team to do it?

**Comment by nedbat** [comment]
@pomegranited if you could make a pull request into open-release/ficus.master, that would be best.

**Pull request merged**